src/control/system/pool.go

```diff
@@ -88,51 +84,35 @@ func (pss *PoolServiceStorage) CreationRanks() []ranklist.Rank {
 	return pss.creationRanks.Ranks()
 }
 
-// CurrentRanks returns the set of target ranks associated
-// with the pool's current.
-func (pss *PoolServiceStorage) CurrentRanks() []ranklist.Rank {
-	pss.Lock()
-	defer pss.Unlock()
-
-	if pss.currentRanks == nil {
-		var err error
-		pss.currentRanks, err = ranklist.CreateRankSet(pss.CurrentRankStr)
-		if err != nil {
-			return nil
-		}
-	}
-	return pss.currentRanks.Ranks()
-}
-
-// TotalSCM returns the total amount of SCM storage allocated to
-// the pool, calculated from the current set of ranks multiplied
-// by the per-rank SCM allocation made at creation time.
-func (pss *PoolServiceStorage) TotalSCM() uint64 {
-	if len(pss.PerRankTierStorage) >= 1 {
-		return uint64(len(pss.CurrentRanks())) * pss.PerRankTierStorage[0]
-	}
-	return 0
-}
-
-// TotalNVMe returns the total amount of NVMe storage allocated to
-// the pool, calculated from the current set of ranks multiplied
-// by the per-rank NVMe allocation made at creation time.
-func (pss *PoolServiceStorage) TotalNVMe() uint64 {
-	if len(pss.PerRankTierStorage) >= 2 {
-		sum := uint64(0)
-		for _, tierStorage := range pss.PerRankTierStorage[1:] {
-			sum += uint64(len(pss.CurrentRanks())) * tierStorage
-		}
-		return sum
-	}
-	return 0
-}
-
-func (pss *PoolServiceStorage) String() string {
-	if pss == nil {
-		return "no pool storage info available"
-	}
-	return fmt.Sprintf("total SCM: %s, total NVMe: %s",
-		humanize.Bytes(pss.TotalSCM()),
-		humanize.Bytes(pss.TotalNVMe()))
-}
+//// TotalSCM returns the total amount of SCM storage allocated to
```

Comment:
Did this need to be removed? If a future PR will re-enable, should add a comment pointing to the ticket.

Reply:
This needs to be removed, I will remove as part of a follow-on PR, thanks.